dapp-hedera-sdk/package.json

```diff
@@ -1,11 +1,18 @@
 {
-    "type": "module",
-    "scripts": {
-        "build": "npm run static && npm run bundle",
-        "static": "cp -r ./static/* ./dist/",
-        "bundle": "bun build ./src/dapp.ts --outdir ./dist --format 'esm' --target 'browser'",
-        "serve": "npx http-server@14.1.1 ./dist -c-1 -p 8111"
-    },
-    "dependencies": {
-    }
+  "type": "module",
+  "scripts": {
+    "dev": "npm run build-vue && npm run static && npm run serve",
+    "static": "cp -r ./static/* ./dist/",
+    "build-bun": "bun build --outdir ./dist --format 'esm' --target 'browser' ./src/dapp.js",
```

Comment:
Is there a need to keep the `build-bun` as it's not being used anymore or is this rather for backward compatibility?

Reply:
The build using bun is *currently* not working due to some issue that has a (long running) WIP fix. Ideally when that is merged, we can provide 2 ways to build the app. So I think of this as a marker/ reminder to come back to this later!